src/data/forecast.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
If the weather data is now coming from an API call this file is no longer required and should be deleted